docs/examples/storage/scaleway/upload_example.py

```diff
@@ -0,0 +1,16 @@
+from libcloud.storage.types import Provider
+from libcloud.storage.providers import get_driver
+
+FILE_PATH = "/home/user/myfile.tar.gz"
+
+cls = get_driver(Provider.SCALEWAY)
+driver = cls("api key", "api secret key", host="s3.fr-par.scw.cloud")
```

Comment:
I believe this example should be updated so a ``region`` instead of ``host`` argument is specified, right?